zest/releaser/pypi.py

```diff
@@ -424,7 +471,7 @@ def create_wheel(self):
             # If the wheel package is not available, we obviously
             # cannot create wheels.
             return False
-        return self._get_boolean("zest.releaser", "create-wheel", True)
+        return self.config.get("create-wheel", True)
```

Comment:
Much nicer that the boolean conversion now happens at read-time, instead of here!